api/backend/ecs/environment_manager.go

```diff
@@ -49,19 +49,17 @@ func (this *ECSEnvironmentManager) ListEnvironments() ([]*models.Environment, er
 		return nil, err
 	}
 
-	models := []*models.Environment{}
-	for _, cluster := range clusters {
+	environments := make([]*models.Environment, len(clusters))
```

Comment:
this is an across the board comment, but I much prefer using context-specific names over `models` as you do here (and throughout) 👍